roles/glance/templates/etc/collectd/plugins/glance.conf

```diff
@@ -1,3 +1,5 @@
+# {{ ansible_managed }}
```

Comment:
My concern with ansible_managed is that it inserts a date/time which sets changed and prompts a service restart. 

But i defer to your preference.